front/src/reducers/osrdconf/stdcmConf/stdcmConfReducers.spec.ts

```diff
@@ -201,58 +222,74 @@ describe('stdcmConfReducers', () => {
   });
 
   describe('StdcmResults updates', () => {
+    const simulationInputs = {
+      pathSteps: stdcmPathSteps,
+      linkedTrains: { anteriorTrain: undefined, posteriorTrain: undefined },
+      consist: {
+        totalMass: 100,
+        totalLength: 50,
+        maxSpeed: 25,
+        speedLimitByTag: 'new-tag',
+      },
+    };
+
     const simulation = {
       index: 0,
       creationDate: new Date(),
-      inputs: {
-        pathSteps: stdcmPathSteps,
-        linkedTrains: { anteriorTrain: undefined, posteriorTrain: undefined },
-      },
+      inputs: simulationInputs,
     };
 
-    it('should handle adding new simulations', () => {
+    it('should create new simulation', () => {
```

Comment:
If you made the other change
```suggestion
    it('should add a new simulation', () => {
```